toxcore/tox.c

```diff
@@ -608,7 +646,7 @@ void tox_kill(Tox *tox)
     }
 
     lock(tox);
-    LOGGER_ASSERT(tox->m->log, tox->m->msi_packet == nullptr, "Attempted to kill tox while toxav is still alive");
+    // ToxAV // LOGGER_ASSERT(tox->m->log, tox->m->msi_packet == nullptr, "Attempted to kill tox while toxav is still alive");
```

Comment:
Is there something we can do to still check this? E.g. check that the callback for the toxav packet isn't set anymore? Yes, it will put knowledge of toxav into toxcore, but only in an assert.

Reply:
done